docs/KernelFeatures/UsingSecureVault.md

```diff
@@ -106,3 +106,63 @@ All the Secret Repository implementations should derive from the Secret Reposito
             return new byte[0];
         }
     }
+```
+
+## Accessing Secure Vault in non OSGI environment.
+
+SecureVault functionality can access in non OSGI mode by following the below steps.
+
+1. Add secure vault dependency to the client project.
+
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.securevault</artifactId>
+            <version>${carbon.kernel.version}</version>
+        </dependency>
+
+2. Add following system properties to the client program.
```

Comment:
Since this is a library, shouldn't this be passed as args when trying to initialize the secure vault instead of setting this as a system property?